rapids_build_backend/impls.py

```diff
@@ -9,11 +9,16 @@
 from importlib import import_module
 
 import tomli_w
-from packaging.requirements import Requirement
-from packaging.specifiers import SpecifierSet
+import yaml
+from rapids_dependency_file_generator.cli import generate_matrix
```

Comment:
The fact that we have to import from the dfg cli module probably indicates that we should be doing some reorganization of dfg, in line with what you've been saying about us needing to stabilize the dfg API.

Reply:
> *... needing to stabilize the dfg API*

I just put up an issue documenting a small (but I think important!) part of that work: https://github.com/rapidsai/dependency-file-generator/issues/76